docs/examples/example.ipynb

```diff
@@ -7,9 +7,10 @@
    "source": [
     "# Example usage\n",
     "\n",
-    "The Viam SDK can be used in two ways:\n",
+    "The Viam SDK can be used in three ways:\n",
     "1. As a client to connect to a (remote or local) robot\n",
-    "2. Integrate custom resources to a robot\n",
+    "2. To create custom components and provide additional functionality to a robot\n",
```

Comment:
This looks like it's reverting some changes that @purplenicole730 made with a recent documentation PR. is that intentional?